scripts/custom_lint_checks.py

```diff
@@ -269,6 +269,14 @@ def visit_functiondef(self, node):
         self.check_functiondef_yields(node, node_doc)
 
     def check_functiondef_params(self, node, node_doc):
+        """Called for checking function definition parameter.
+
+        Args:
+            node: astroid.scoped_nodes.Function. Node for a function or
+                method definition in the AST.
+            node_doc: docstring.
```

Comment:
I think the type is wrong. Please do a `type(node_doc)` to know the exact type.

Reply:
Here and everywhere below.

Reply:
Where do I do this type(node_doc) to find out the type.

Reply:
You can do this within the function body itself.